Modules/Role/Http/Services/RoleService.php

```diff
@@ -218,6 +237,10 @@ public function postUser(array $requestData): array
         $user->role = $role;
         $user->token = '';
         $user->save();
+        $user = User::findOrFail($user->id);
```

Comment:
Can't we use the $user object defined above? do we have to find the user again using this query?
If it's not necessary, then please remove it.

Reply:
Fixed